packages/spanner/src/admin/database/v1/database_admin_client.js

```diff
@@ -30,7 +30,6 @@
 var configData = require('./database_admin_client_config');
 var extend = require('extend');
 var gax = require('google-gax');
-var path = require('path');
```

Comment:
it was added manually to load a non public proto files and probably I've forgotten to remove them; no need to fix on the code generator, but that sounds like we should progress more automation around this area.